packages/kbn-esql-ast/src/ast_helpers.ts

```diff
@@ -275,12 +275,21 @@ function safeBackticksRemoval(text: string | undefined) {
   return text?.replace(TICKS_REGEX, '').replace(DOUBLE_TICKS_REGEX, SINGLE_BACKTICK) || '';
 }
 
+function safeSingleQuoteRemoval(text: string | undefined) {
+  return text?.replace(/"/g, '') || '';
+}
+
+function safeTripleQuoteRemoval(text: string | undefined) {
+  return text?.replace(/"""/g, '') || '';
+}
+
 export function sanitizeIdentifierString(ctx: ParserRuleContext) {
   const result =
     getUnquotedText(ctx)?.getText() ||
+    safeSingleQuoteRemoval(ctx.getText()) ||
```

Comment:
I guess here we can just safeSingleQuoteRemoval, as it will replace any triple quote as well. Should we safe guard against a double quote like `""from foo-{{dd}}""`?